src/device-registry/controllers/create-activity.js

```diff
@@ -0,0 +1,357 @@
+const HTTPStatus = require("http-status");
+const isEmpty = require("is-empty");
```

Comment:
a new controller to handle the site activities use case. All part of refactoring.